daemon/src/bitmex_price_feed.rs

```diff
@@ -9,6 +8,7 @@ use rust_decimal::Decimal;
 use std::convert::TryFrom;
 use std::time::Duration;
 use time::OffsetDateTime;
+use tokio_tasks::Tasks;
```

Comment:
👍 Interesting that these use statements are available from `crate`. There are a lot of `use crate::Tasks` in this crate :(

Reply:
I'll see to pack them all in this commit!

Reply:
Thanks!